modules/Emails/EmailsController.php

```diff
@@ -449,6 +450,24 @@ public function action_GetFromFields()
             }
         }
 
+        if ($sugar_config['email_allow_send_as_user']) {
+            $data[] = array(
+                'type' => 'personal',
+                'id' => $current_user->id,
+                'attributes' => array(
+                    'from' => $current_user->email1,
+                    'name' => $current_user->full_name,
+                ),
+                'prepend' => $prependSignature,
+                'isPersonalEmailAccount' => true,
+                'isGroupEmailAccount' => false,
+                'emailSignatures' => array(
+                    'html' => utf8_encode(html_entity_decode($defaultEmailSignature['signature_html'])),
+                    'plain' => $defaultEmailSignature['signature'],
```

Comment:
`$defaultEmailSignature['signature']` could be unset. can you please check if it's exists? (see above)

Reply:
I'll correct this.

Reply:
After looking at the code, I realized that the check for an empty signature is taken care of on line 385. If the signature is empty it's set to:
if (empty($defaultEmailSignature)) {
            $defaultEmailSignature = array(
                'html' => '<br>',
                'plain' => '\r\n',
            );